File: Loop.cs
using System;
class Loop {
    static void Main() {
        for (int i = 1; i <= 5; i++) Console.WriteLine(i);
        int s = 0, j = 1;
        while (j <= 5) { s += j; j++; }
        Console.WriteLine($"Soma = {s}");
        int k = 3;
        do { Console.WriteLine($"k = {k}"); k--; } while (k > 0);
    }
}

File: loop.cpp
#include <iostream>
using namespace std;
int main() {
    for (int i = 1; i <= 5; ++i) cout << i << endl;
    int s = 0, j = 1;
    while (j <= 5) { s += j; j++; }
    cout << "Soma = " << s << endl;
    int k = 3;
    do { cout << "k = " << k << endl; k--; } while (k > 0);
    return 0;
}